share/plugins/tuning/tuning.qml

```diff
@@ -739,115 +739,104 @@ MuseScore {
     Item {
         anchors.fill: parent
 
+        ButtonGroup { id: tempamentTypeGroup }
+
+        component TuningItem: RadioButton {
+            padding: 4
```

Comment:
should it be 2 like in other places?

Reply:
no, there are fewer elements here, it looks better